src/main/java/de/koudingspawn/vault/kubernetes/KubernetesService.java

```diff
@@ -98,6 +100,21 @@ private ObjectMeta metaData(ObjectMeta resource, String compare) {
         annotations.put(crdName + LAST_UPDATE_ANNOTATION, LocalDateTime.now().toString());
         annotations.put(crdName + COMPARE_ANNOTATION, compare);
         meta.setAnnotations(annotations);
+
+        boolean blockOwnerDeletion = false;
+        boolean controller = true;
+        OwnerReference owner = new OwnerReference(
+          "koudingspawn.de/v1",
```

Comment:
It's a bit hacky, but can you please have a look here: https://github.com/DaspawnW/vault-crd/blob/master/src/main/java/de/koudingspawn/vault/kubernetes/KubernetesConnection.java#L48
This allows to change the name of the domain.

Reply:
I think I follow you... addressed in https://github.com/DaspawnW/vault-crd/pull/51/commits/5f5f9fe83b8747e74009e20f700f94bc3a7331d2